lib/index.coffee

```diff
@@ -38,8 +37,5 @@ abstract_mapper = (name, direction) ->
 
   return res or name
 
-name_to_adapter = (name) ->
-  abstract_mapper(name, 'left')
-
```

Comment:
it seems like you've eliminated mapping in this direction completely, could you eliminate the extra arg in `abstract_mapper` and perhaps remove this extra `name_to_adapter` function as well, or do you still wanna keep this flexibility.